02-express-tutorial/new-public/practiceExpress.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express')
+const path = require('path')
+
+const app = express()
+
+
+app.get('/', (req,res) =>{
+    res.sendFile(path.resolve(__dirname, './new-public/index.html'))
```

Comment:
Alternatively, if you use Express.static(path) middleware, the default index page for "./" api is the index.html in that path.   What you have done also works. Good work.